xml/System/Type.xml

```diff
@@ -4209,9 +4209,26 @@ In .NET 6 and earlier versions, the <xref:System.Type.GetFields%2A> method does
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>When overridden in a derived class, returns the calling conventions of the current function pointer <see cref="T:System.Type" />.</summary>
+        <returns>An array of <see cref="T:System.Type" /> objects representing all the calling conventions for the current function pointer <see cref="T:System.Type" />.
+  
+ -or-  
+  
+ An empty array of type <see cref="T:System.Type" />, if no calling conventions are defined for the current function pointer <see cref="T:System.Type" />.
+
+ -or-  
+  
+ An empty array of type <see cref="T:System.Type" />, if the current function pointer <see cref="T:System.Type" /> is not a modified <see cref="T:System.Type" />. A modified <see cref="T:System.Type" /> is obtained from <see cref="M:System.Reflection.FieldInfo.GetModifiedFieldType" />, <see cref="M:System.Reflection.PropertyInfo.GetModifiedPropertyType" /> or <see cref="M:System.Reflection.ParameterInfo.GetModifiedParameterType" />
+        </returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <xref:System.Runtime.CompilerServices> namespace.  
+  
+ ]]></format>
+        </remarks>      
+        <exception cref="T:System.InvalidOperationException">The current <see cref="T:System.Type" /> object is not a function pointer. That is, the <see cref="P:System.Type.IsFunctionPointer" /> property returns <see langword="false" />.</exception>
```

Comment:
NIT: It seems to me better to start with `When` or `If`, up to you
```suggestion
        <exception cref="T:System.InvalidOperationException">If the current <see cref="T:System.Type" /> object is not a function pointer. That is, the <see cref="P:System.Type.IsFunctionPointer" /> property returns <see langword="false" />.</exception>
```

Reply:
The existing usages of "InvalidOperationException" in similar cases don't use "If" so I kept that however the link to Type here is somewhat redundant so I removed it.